src/components/Authentication/LoginPage.js

```diff
@@ -90,14 +90,17 @@ const LoginPage = () => {
         setHasError(false);
     }, []);
 
-    const handleLogin = useCallback((event) => {
+    const handleLogin = useCallback(async (event) => {
         event.preventDefault();
-        login({
+        const loginResponse = await login({
             "variables": {
                 password,
                 "username": email,
             },
         });
+        if (loginResponse?.data?.tokenAuth) {
+            history.push("/")
+        }
```

Comment:
Are we sure that redux auth state is fully loaded when the redirect occurs?

Reply:
it looks like it - i'm not getting any errors when i was testing this